unischedule/android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     namespace "com.example.unischedule"
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 34
```

Comment:
For the uso of local notifications I had to change this. What you think guys? @iGotty @jsurrea

Reply:
LGTM 👍